ps_buybuttonlite.php

```diff
@@ -0,0 +1,197 @@
+    {
+        $tab = new Tab();
+        $tab->class_name = $this->controllerAdmin;
+        $tab->active = 1;
+        $tab->name = array();
+        foreach (Language::getLanguages(true) as $lang) {
+            $tab->name[$lang['id_lang']] = $this->name;
+        }
+        $tab->id_parent = -1;
+        $tab->module = $this->name;
+
+        return $tab->add();
+    }
+
+    /**
+     * Unregister admin controller
+     *
+     * @param none
+     * @return bool
+     */
+    public function uninstallTab()
+    {
+        $id_tab = (int)Tab::getIdFromClassName($this->controllerAdmin);
+        $tab = new Tab($id_tab);
+        if (Validate::isLoadedObject($tab)) {
+            return ($tab->delete());
+        } else {
+            return false;
+        }
+    }
+
+    /**
+     * Load dependencies
+     *
+     * @param none
+     * @return bool
+     */
+    public function loadAsset()
+    {
+        $css = array(
+            'https://unpkg.com/element-ui/lib/theme-chalk/index.css',
+            $this->css_path.'override-element-ui.css',
+            $this->css_path.'back.css',
+        );
+
+        return $this->context->controller->addCSS($css, 'all');
+    }
+
+    /**
+     * Load the configuration form
+     */
+    public function getContent()
+    {
+        $this->loadAsset();
+
+        $link = new Link();
+        $adminAjaxController = $link->getAdminLink($this->controllerAdmin);
+
+        $availableTranckingLanguage = array('en', 'fr', 'es', 'it', 'de', 'nl', 'pt', 'pl', 'ru');
+
+        $iso_code = 'en';
+        if (in_array($this->context->language->iso_code, $availableTranckingLanguage)) {
+            $iso_code = $this->context->language->iso_code;
+        }
+        $trackingAddons = 'https://addons.prestashop.com/'.$iso_code.'/blog-forum-actualites/41139-buy-button.html?utm_source=back-office&utm_medium=native-module&utm_campaign=back-office-'.strtoupper($iso_code).'&utm_content=Permalink';
+
+        $confTranslations = array(
+            'selectProduct' => $this->trans('Select Product', array(), 'Modules.Buybuttonlite.Admin'),
+            'searchProduct' => $this->trans('Search Product', array(), 'Modules.Buybuttonlite.Admin'),
+            'action' => $this->trans('Select action', array(), 'Modules.Buybuttonlite.Admin'),
+            'sharableLink' => $this->trans('Get sharable link', array(), 'Modules.Buybuttonlite.Admin'),
+            'copyToClipboard' => $this->trans('Copy to clipboard', array(), 'Modules.Buybuttonlite.Admin'),
+            'linkCopied' => $this->trans('Link copied to clipboard', array(), 'Modules.Buybuttonlite.Admin'),
+        );
+
+        $bannerPromoTranslations = array(
+            'copyToClipboard' => $this->trans('Copy to clipboard', array(), 'Modules.Buybuttonlite.Admin'),
+            'discover' => $this->trans('Discover', array(), 'Modules.Buybuttonlite.Admin'),
+            'screenshots' => $this->trans('Screenshots', array(), 'Modules.Buybuttonlite.Admin'),
```

Comment:
Currently, the wording does not exist in the Crowdin catalogs but as it may be considered as a generic term, perhaps we should localize this translation key in the _Admin.Global_ file so that it can be reused elsewhere in the back office. What do you think?

Reply:
Yes, it should be useful. Added to Admin.Global :)

Reply:
Sorry, I was talking about _Screenshots_, not _Copy to clipboard_ which is well localized.